packages/client/tests/functional/metrics/enabled/tests.ts

```diff
@@ -470,7 +474,13 @@ testMatrix.setupTestSuite(
         const metrics = await prisma.$metrics.prometheus()
         expect((metrics.match(/prisma_client_queries_total \d/g) || []).length).toBe(1)
         expect((metrics.match(/prisma_client_queries_active \d/g) || []).length).toBe(1)
-        expect((metrics.match(/prisma_client_queries_wait \d/g) || []).length).toBe(1)
+
+        // Our test suite shows that the value can be 0 sometimes
+        // Last seen in `Tests / Client func&legacy-notypes (4/5, library, 20, relationJoins)` run for SQLite
+        // And also for Postgres, see below
+        // https://github.com/prisma/prisma/issues/13579#issuecomment-1794323813
+        // @ts-expect-error - needed to make typechecking pass when running the test at the moment
+        expect((metrics.match(/prisma_client_queries_wait \d/g) || []).length).toBeOneOf([0, 1])
```

Comment:
Sounds like I should create a tech debt issue to figure this one out another day